AutoTenez.py

```diff
@@ -137,13 +139,54 @@ def make_reservation(headers, cookies, date_tomorrow, md5slotkey, your_external_
     print("ERROR! Enter your email address and password")
     sys.exit(0)
 
+# Parse input arguments
+parser = argparse.ArgumentParser(description='Reserve tennis court for tomorrow.')
+parser.add_argument('-c',  '--courts',               nargs='+', help='Specify courts ("Baan X", where X is the court number). Default setting is all courts.')
+parser.add_argument('-d',  '--date',                            help='Specify the date to make the reservation (yyyy-mm-dd)')
+parser.add_argument('-t2', '--time_second_choice',   nargs='+', help='Time you would like to reserve for the second option (hh:mm).')
+parser.add_argument('-c2', '--courts_second_choice', nargs='+', help='Specify courts for the second option.')
```

Comment:
Copy explanation from `--time` and `--courts`. By including "One or two consecutive times are allowed (hh:mm)" and "("Baan X", where X is the court number)".